samples/iolib/src/main/cpp/player/SimpleMultiPlayer.h

```diff
@@ -60,16 +58,16 @@ class SimpleMultiPlayer : public AudioStreamCallback  {
 
 private:
     // Oboe Audio Stream
-    AudioStream *mAudioStream { nullptr };
+    oboe::AudioStream *mAudioStream { nullptr };
```

Comment:
You can use a `ManagedStream` here to avoid the raw pointer

Reply:
Done. (I think)